resources/campaigns/campaigns.yml

```diff
@@ -0,0 +1,88 @@
+get:
+  operationId: campaigns_list
+
+  summary: List
+
+  description: >-
+    Returns a list of your campaigns. The campaigns are returned sorted by
+    creation date, with the most recently created campaigns appearing first.
+
+  tags:
+    - Campaigns
+
+  parameters:
+    - $ref: "../../shared/parameters/limit.yml"
+    - $ref: "../../shared/parameters/include.yml"
+    - $ref: "../../shared/parameters/before_after.yml"
+
+  responses:
+    "200":
+      $ref: "responses/all_campaigns.yml"
+
+  x-codeSamples:
+    - lang: Shell
+      source: |
+        curl https://api.lob.com/v1/campaigns \
+          -u <YOUR API KEY>:
+      label: CURL
+
+post:
+  operationId: campaign_create
+
+  summary: Create
+
+  description: >-
+    Creates a new campaign with the provided properties
+
+  tags:
+    - Campaigns
+
+  parameters:
+    - $ref: "../../shared/parameters/lang_spec.yml"
+
+  requestBody:
+    required: true
+    content:
+      application/json:
+        schema:
+          $ref: "models/campaign_writable.yml"
+        example:
+          description: "Demo Campaign"
+          # complete example using all create payload properties
+
+      application/x-www-form-urlencoded:
+        schema:
+          $ref: "models/campaign_writable.yml"
+        encoding: # be sure to check which encodings apply
+          example-prop:
+            style: deepObject
+            explode: true
+        example:
+          description: encoded demo
+          # complete example using all create payload properties
```

Comment:
Same question as above, we should either have examples or add a TODO with a JIRA link so it's not lost.